### DISCUSSION TITLE
How do you change axis on GRBL into mm?

### DISCUSSION THREAD

**Discussion by karlwoood** [arkypita/LaserGRBL#1390] [Q&A]
Hi, I have 2 pc systems one running a 3018 extended to a 3036 and one running a 6550 both using laser GRBL software but although the settings are the same on both systems only the 3036 does projects are in mm I m not sure what the other one is in I thought it was cm but now I'm not sure if it inches. I've tried changing the $13 to 1 and 0 but neither make a difference to the axis measurements I can't seem to get them into mm, probably something really simple but I can't seem to figure it out. Can anyone help me please, thanks

**Comment by gmmanonymus111** [comment]
You can't really change it to mm or cm or inches. The `$13` command only reports the measurements in inches instead of mm, but your movement is the same. Take a look at `$100` & `$101`, as those variables set your travel resolution. When set incorrectly, your machine will draw shorter/longer lines than specified in your GCode. Leadscrew driven machines usually use higher resolutin than belt driven ones. I suppose your '3018' is leadscrew driven and the '6550' is belt driven, an you used the same config on both machines. 
You can [take a look here](https://learn.sparkfun.com/tutorials/stepoko-powered-by-grbl-hookup-guide/firmware-configuring-grbl-and-calibrating) to try to calibrate your travel resolution.
